cheats/gpg.cheat

```diff
@@ -0,0 +1,39 @@
+% gpg
+
+# gpg version
+gpg --version
+
+# gpg generate key
+gpg --gen-key
+
+# list keys
+gpg --list-keys
+
+# distribute public key to key server
+gpg --keyserver hkp://pool.sks-keyservers.net --send-keys <public-key>
```

Comment:
What is this for? 🤔

Reply:
replaced the hardcoded public key server for a variable